EventsExpress/ClientApp/src/actions/events/filter/actions.js

```diff
@@ -1,14 +1,22 @@
 import { stringify, exclude } from 'query-string';
 import { get_events } from '../../event/event-list-action';
+import { enumLocationType } from '../../../constants/EventLocationType';
 
 export const applyFilters = filters => {
     return async dispatch => {
         filters.owners = filters?.organizers?.map(organizer => organizer.id);
-
+        filters.locationtype = filters?.location.type;
```

Comment:
Are you sure, that it should be `locationtype` not `locationType`? Would this property be bound properly?

Reply:
yes it works so (locationtype)